core/modules/widgets/browse.js

```diff
@@ -91,10 +96,27 @@ BrowseWidget.prototype.execute = function() {
 	this.nwdirectory = this.getAttribute("nwdirectory");
 };
 
+BrowseWidget.prototype.assignDomNodeClasses = function() {
```

Comment:
I wonder if we might make this a method in widget.js. It would need to be refactored to to take an options argument with a few options:

* an array of classnames to be added to those specified in the attribute
* an override of the name of the attribute, if we have any widgets that use a non-standard attribute name (I can't remember if we do)

Reply:
@Jermolene , I've refactored this method into widget.js
Now it accepts an array of classnames to be added to those specified in the attribute
As for the override of the name of the attribute I need some advice, I don't know what's meant with this